test/relative-time.js

```diff
@@ -16,80 +16,115 @@ describe("relative-time", function() {
+      expect(relativeTime.format(new Date("2016-04-11 05:00"))).to.equal("in 6 hours");
+      expect(relativeTime.format(new Date("2016-04-11 06:00"))).to.equal("tomorrow");
+
+      sinon.useFakeTimers(new Date("2016-01-31 23:00").getTime());
+      expect(relativeTime.format(new Date("2016-02-01 05:00"))).to.equal("in 6 hours");
+      expect(relativeTime.format(new Date("2016-02-01 07:00"))).to.equal("tomorrow");
+
+      sinon.useFakeTimers(new Date("2016-12-31 23:00").getTime());
+      expect(relativeTime.format(new Date("2017-01-01 05:00"))).to.equal("in 6 hours");
+      expect(relativeTime.format(new Date("2017-01-01 07:00"))).to.equal("tomorrow");
+    });
+
+    it("should format days-distant dates", function() {
+      expect(relativeTime.format(new Date("2016-04-01 00:00"))).to.equal("9 days ago");
+      expect(relativeTime.format(new Date("2016-04-09 18:00"))).to.equal("yesterday");
+      expect(relativeTime.format(new Date("2016-04-11 09:00"))).to.equal("tomorrow");
+      expect(relativeTime.format(new Date("2016-04-30 23:59"))).to.equal("in 20 days");
+      expect(relativeTime.format(new Date("2016-03-31 23:59"))).to.equal("last month");
+      expect(relativeTime.format(new Date("2016-05-01 00:00"))).to.equal("next month");
+
+      sinon.useFakeTimers(new Date("2016-04-06 12:00").getTime());
+      expect(relativeTime.format(new Date("2016-03-31 23:59"))).to.equal("6 days ago");
+
+      sinon.useFakeTimers(new Date("2016-04-25 23:00").getTime());
+      expect(relativeTime.format(new Date("2016-05-01 00:00"))).to.equal("in 6 days");
+    });
+
+    it("should format months-distant dates", function() {
+      expect(relativeTime.format(new Date("2016-01-01 00:00"))).to.equal("3 months ago");
+      expect(relativeTime.format(new Date("2016-03-01 00:00"))).to.equal("last month");
+      expect(relativeTime.format(new Date("2016-05-01 00:00"))).to.equal("next month");
+      expect(relativeTime.format(new Date("2016-11-31 23:59"))).to.equal("in 8 months");
+
+      sinon.useFakeTimers(new Date("2017-01-12 18:30").getTime());
+      expect(relativeTime.format(new Date("2016-12-29 18:30"))).to.equal("last month");
+
+      sinon.useFakeTimers(new Date("2016-12-29 18:30").getTime());
+      expect(relativeTime.format(new Date("2017-01-12 18:30"))).to.equal("next month");
+
+      sinon.useFakeTimers(new Date("2016-02-28 12:00").getTime());
+      expect(relativeTime.format(new Date("2015-12-31 23:59"))).to.equal("2 months ago");
+    });
+
+    it("should format years-distant dates", function() {
+      expect(relativeTime.format(new Date("2010-06-01 12:00"))).to.equal("6 years ago");
+      expect(relativeTime.format(new Date("2015-12-31 23:59"))).to.equal("last year");
+      expect(relativeTime.format(new Date("2017-01-01 00:00"))).to.equal("next year");
+
+      sinon.useFakeTimers(new Date("2016-10-02 12:00").getTime());
+      expect(relativeTime.format(new Date("2017-01-01 00:00"))).to.equal("next year");
+    });
```

Comment:
This is best reviewed by ignoring white-space change... Note the only actual change was the "best-fit" describe and this corrected test:

```diff
- expect(relativeTime.format(new Date("2016-04-10 11:00:01"))).to.equal("59 minutes ago");
+ expect(relativeTime.format(new Date("2016-04-10 11:01:00"))).to.equal("59 minutes ago");
```